### ISSUE TITLE
Client Settings - Service blocking Time Zone fails

### ISSUE THREAD

**Issue by thewonderer73** [AdguardTeam/AdGuardHome#6565]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Linux, ARMv7

### Installation

Custom package (OpenWrt, HomeAssistant, etc; please mention in the description)

### Setup

On a router, DHCP is handled by the router

### AdGuard Home version

v0.107.43

### Action

Adguard Home, Settings, Client Settings,  Add Client (or Edit existing client).  Add IP address of device. Device has MAC address set to static IP address in GL-iNET MT6000 (Flint2).

Select 'Block Specific Service' and choose just YouTube.  Go to 'Pause service blocking' and select a time and time zone . Click save and error appears.

Delete Client and try again.  Still same error message.

### Expected result

Pause service blocking saves selected days and time successfully for Oceania / Brisbane.  Blocked YouTube service is unblocked during the time specified.

### Actual result

Error: control/clients/update | failed to process request body: unknown time zone Australia/Brisbane | 4000

**Comment by ainar-g** [comment]
We cannot reproduce this, unfortunately.

> failed to process request body: unknown time zone Australia/Brisbane

This likely means that your machine doesn't have that time zone installed.  Please update the router's `tzdata` package, if you can.

I've also noticed that the list doesn't include a list of fixed timezones, which could help work around these issues, and we do need to add them.  I've filed #6568 for that.

**Comment by jdawgzim** [comment]
I had same issue and this is how I fixed it:

1) ssh-ed to my Raspberry Pi and checked date with "date" command
2) It showed the wrong time zone
3) used "sudo raspi-config" to localize the time zone to where I live
4) Select Finish (had to hit TAB key to be able to select Finish)
5) "sudo reboot"

It's weird how inside AdGuard all showed the correct timestamps for events but it seems for this feature it goes to the OS for the time.

**Comment by thewonderer73** [comment]
Thanks.  I just SSH into my gl-inet Flint2 and checked the 'date'  It's set correctly...  Sat Dec 30 10:22:50 AEST 2023

**Comment by ainar-g** [comment]
@odcold, see the messages above.  First and foremost, update your tzdata, if you're able to.  If not, you're probably stuck using `Etc/` zones.

If you want to investigate further, the verbose logs are the way.  I'm not sure what you mean by “only error 400” though.  You should be able to see records with `[debug]` prefix in your service logs.

**Comment by ainar-g** [comment]
I feel like at this point it's easier to just not rely on the operating system's own tzdata at all.  Go allows shipping a recent version of tzdata with the binary, and we'll probably go this way.  I'll file a separate issue about that.

**Comment by Orel-A** [comment]
Anyone found a solution for this? Any quick fix for it?
